src/Streaming/Internal.hs

```diff
@@ -962,6 +962,15 @@ unseparate str = destroyExposed
   return
 {-#INLINABLE unseparate #-}
 
+-- | @copy = expand extend@, where @extend@ is from Control.Comonad
```

Comment:
This semi-documentation is a bit of a lie, because there isn't actually a `Comonad (Of a)` instance. But there could be one!